data/src/main/java/com/hous/data/api/HomeApi.kt

```diff
@@ -1,51 +1,51 @@
-package com.hous.hous_aos.data.api
-
-import com.hous.hous_aos.data.entity.Event
-import com.hous.hous_aos.data.entity.Homie
-import com.hous.hous_aos.data.entity.ResultData
-import com.hous.hous_aos.data.model.WrapperClass
-import com.hous.hous_aos.data.model.request.EventListRequest
-import com.hous.hous_aos.data.model.response.HomeResponse
+package com.hous.data.api
+
+import com.hous.data.entity.Event
+import com.hous.data.entity.Homie
+import com.hous.data.entity.ResultData
+import com.hous.data.model.WrapperClass
+import com.hous.data.model.request.EventListRequest
+import com.hous.data.model.response.HomeResponse
 import retrofit2.http.*
 
 interface HomeApi {
     @GET("room/{roomId}/home")
     suspend fun getHomeList(
         @Path("roomId") roomId: String
-    ): WrapperClass<HomeResponse>
+    ): com.hous.data.model.WrapperClass<com.hous.data.model.response.HomeResponse>
```

Comment:
우와... 이렇게 쓰는 거군요..

Reply:
```suggestion
    ): WrapperClass<HomeResponse>
```
ㅠㅠ...이건 모듈로 옮기면서 생긴 것... 후에 리팩하면서 요렇게 바꿔주세요..!

Reply:
> 

ㅋㅋㅋㅋ넹~~